providers/unleash/src/test/resources/features.json

```diff
@@ -0,0 +1,4509 @@
+{
+  "version": 1,
+  "features": [
+    {
+      "name": "083db41c-c6c8-427b-815b-f735c3c6da60",
```

Comment:
this file looks very strange and is very long. It is usually better to a nicely crafted examples with descriptions on the purpose of each configuration examples and what we are testing for.

Reply:
It is taken from [Unleash demo](https://github.com/Unleash/unleash-client-go/blob/88862d88f5813e6869f4790da689a1d492b587a6/demo_app_toggles.json#L5). I can make it shorter, yes.